README.md

```diff
@@ -62,3 +62,37 @@ RKE2_URL=https://myserver:6443 RKE2_TOKEN=XXX ./install.sh
 ## Automated deployment
 
 We provide a simple automated way to install RKE2 on AWS via terraform scripts, this method requires terraform to be installed and access to AWS cloud, to get started please checkout the [rke2-build](https://github.com/rancher/rke2-build) repo.
+
+## Configuration File
+
+In addition to configuring RKE2 with environment variables and cli arguments, RKE2 can also use a config file.
+
+By default, values present in a `yaml` file located at `/etc/rancher/rke2/flags.conf` will be used on install.
+
+An example of a basic `server` config file is below:
+
+```yaml
+# /etc/rancher/rke2/flags.conf
+write-kubeconfig-mode: "0644"
+tls-san:
+  - "foo.local"
+node-label:
+  - "foo=bar"
+  - "something=amazing"
+```
+
+In general, cli arguments map to their respective yaml key, with set cli args being represented as yaml lists.
```

Comment:
'set cli args' is a bit ambiguous - could either be read to mean args that you have set, or the set datatype. Maybe say 'repeatable' or 'additive'?

Reply:
That's a good point, will change it to say "repeatable"